docs/sources/clients/promtail/stages/limit.md

```diff
@@ -56,3 +63,18 @@ Given the pipeline:
 ```
 
 Would throttle any log line and drop logs when rate limit.
+
+#### Ratelimit by a label
```

Comment:
I think it's worth mentioning what is the difference between these two.
```
- limit:
    rate: 10
    burst: 10
    drop: true
    by_label_name: "namespace"
```
with this

```
match:
  selector: '{namespace="foo"}"'
  stages:
  - limit:
      rate: 5
      drop: true
match:
  selector: '{namespace!="foo"}"'
  stages:
  - limit:
      rate: 100
      drop: false
```

Like @liguozhong already mentioned, even my first thought would be to use latter config instead of this new config.

I see you explained, with `by_label_name` we can get separate ratelimiter per label. 

Wondering what advantage does it serve?